firefly_client/firefly_client.py

```diff
@@ -25,7 +25,9 @@
 
 _my_localurl = 'http://localhost:8080/firefly'
```

Comment:
Is this actually something that should be in the released code?  I think it would be better for the fallback default to be an exception or other error.  The likelihood in the open-source context that someone other than one of our devs would be running locally on the 8080 port seems really very low, and the error message from the localhost:8080 fallback seems like it is inevitably going to be less understandable to a user -- something like "can't reach Firefly server at localhost:8080" than an explicit "No Firefly server address could be determined for the user's environment." or equivalent.  The latter message is also going to be easier for _us_ to put in context when someone posts it on c.l.o or on Slack.

Reply:
A one time we were planning more local installs so thIs made sense.

Reply:
We can change it in a future PR.

Reply:
Perhaps what we need for that scenario (which I still find interesting, but not specifically in the LSST context) is a little startup-script wrapper for the standalone environment that sets the FIREFLY_URL environment variable before launching the server?

Or something even smarter, once we take a look at David's issues with the conflict between inside-Docker and outside-Docker local URLs...?